### DISCUSSION TITLE
[typo] Error link in readme

### DISCUSSION THREAD

**Discussion by David-Maran** [familytree365/genealogy#1803] [Q&A] [answered]
The link in readme demo section:

```html
https://www.familytree365.com-
````
will be
```html
https://www.familytree365.com
````

I thought the site was closed .... 
And
Creating an account to try the demo is a repellant (that's just my opinion)

**Comment by curtisdelicata** [comment, marked as answer]
Thank you for your report. The reason the demo is registrations is to show how the code works but we may create a demo account to use.

**Reply by alcyoneus-g** [reply]
@everyone you got scammed, please check @curtisdelicata 's Linkedin profile. He is fraud and liar. He never pay for developers. 

He runs fake business called FamilyTree365. He lies developers to contribute the project for a week and pretend to pay after a week but he disappears after receiving PR.

I should alert other honest developers and customers that you should never support Curtis Delicata since he is a fraud.

I can share you more evidence if anybody interested.

https://www.linkedin.com/feed/update/urn:li:activity:6906662303572606976/

![image](https://user-images.githubusercontent.com/72995735/157096643-3d6dd048-5dcf-47ca-b5c4-f97d4064134b.png)

**Comment by alcyoneus-g** [comment]
@everyone you got scammed, please check @curtisdelicata 's Linkedin profile. He is fraud and liar. He never pay for developers. 

He runs fake business called FamilyTree365. He lies developers to contribute the project for a week and pretend to pay after a week but he disappears after receiving PR.

I should alert other honest developers and customers that you should never support Curtis Delicata since he is a fraud.

I can share you more evidence if anybody interested.

https://www.linkedin.com/feed/update/urn:li:activity:6906662303572606976/

![image](https://user-images.githubusercontent.com/72995735/157096643-3d6dd048-5dcf-47ca-b5c4-f97d4064134b.png)
